datatypes/src/collections/feature_collection.rs

```diff
@@ -127,6 +128,17 @@ pub trait FeatureCollectionModifications {
     /// This method fails if the columns do not match
     ///
     fn append(&self, other: &Self) -> Result<Self::Output>;
+
+    /// Rename column `old_column_name` to `new_column_name`.
+    fn rename_column(&self, old_column_name: &str, new_column_name: &str) -> Result<Self::Output> {
+        self.rename_columns(&[(old_column_name, new_column_name)])
+    }
+
+    /// Rename columns with (from, to) tuples.
```

Comment:
please specify whether all columns have to be renamed or only some of them

Reply:
Why should one expect that only part of the inputs is incorporated?

Reply:
I meant that it is not clear to the user of the function whether they have to specify a (from, to) pair for each existing column. Suggestion: "Rename selected columns with (from, to) tuples"

Reply:
e8a91de